tssc/step_implementers/tag_source/git.py

```diff
@@ -1,13 +1,18 @@
 """
 Step Implementer for the tag-source step for Git.
 """
-
+import sh
 from tssc import TSSCFactory
 from tssc import StepImplementer
 from tssc import DefaultSteps
 
 DEFAULT_ARGS = {}
 
+OPTIONAL_ARGS = {
+    'username': None,
+    'password': None
+}
+
 class Git(StepImplementer):
     """
     StepImplementer for the tag-source step for Git.
```

Comment:
add comment about why using sh.git rather then git lbirary.

Reply:
Completed per request. Thank you!